baanders2021reproducibility/openreview/references.bib

```diff
@@ -0,0 +1,30 @@
+@InProceedings{Li_2020_CVPR,
+author = {Li, Peizhao and Zhao, Han and Liu, Hongfu},
+title = {Deep Fair Clustering for Visual Learning},
+booktitle = {The IEEE/CVF Conference on Computer Vision and Pattern Recognition (CVPR)},
+month = {June},
+year = {2020}
+}
+
+@article{Cauchy_Schwarz_divergence,
+author = {Jenssen, Robert and Principe, Jose C. and Erdogmus, Denis and Eltoft, Torbj{\o}rn},
+title = {The Cauchy-Schwarz divergence and Parzen windowing: Connections to graph theory and Mercer kernels},
+journal = {Journal of the Franklin Institute},
+volume = {343},
+number = {6},
+pages = {614-629},
+year = {2006},
+month = {September}
+}
+
+@misc{git,
+    author    = {Li, Peizhao and Zhao, Han and Liu, Hongfu},
+    title     = {Deep Fair Clustering},
+    url       = {https://github.com/brandeis-machine-learning/DeepFairClustering}
+}
+
+@misc{onze_git,
+    author    = {Al Gerges, Chris and Baanders, Pauline and Reints, Nienke and Teule, Tobias},
+    title     = {Reproducing Deep Fair Clustering},
+    url       = {https://github.com/topteulen/UVA-FACT}
+}
```

Comment:
Please copy these references in `bibliography.bib` file in journal folder for compilation.